hugegraph-dist/src/main/java/com/baidu/hugegraph/dist/HugeGraphServer.java

```diff
@@ -90,18 +106,20 @@ public void stop() {
 
     public static void main(String[] args) throws Exception {
         if (args.length != 2) {
-            String msg = "HugeGraphServer can only accept two config files";
+            String msg = "Start HugeGraphServer need to pass 2 parameters, " +
+                         "they are the config files of GremlinServer and " +
+                         "RestServer, for example: conf/gremlin-server.yaml " +
+                         "conf/rest-server.properties";
             LOG.error(msg);
             throw new HugeException(msg);
         }
 
-        HugeRestServer.register();
+        HugeGraphServer.register();
 
         HugeGraphServer server = new HugeGraphServer(args[0], args[1]);
         Runtime.getRuntime().addShutdownHook(new Thread(() -> {
             LOG.info("HugeGraphServer stopping");
             server.stop();
-            LOG.info("HugeGraphServer stopped");
```

Comment:
sure to remove it?

Reply:
not sure, just feel the logs are a bit redundant